src/app/app.component.ts

```diff
@@ -1,4 +1,5 @@
 import { Component } from '@angular/core';
+import { AlmDropdown, AlmDropdownToggle } from './shared-component/alm-dropdown/alm-dropdown';
```

Comment:
- Where are these files?
- We are using **pf** instead of **alm** for any independent component's prefix.